### PR TITLE
Audioplayback scrubber

### PR DESCRIPTION
**Pull request by JB184351**
I thought I was having some bugs where my audioState would change to stop while scrubbing but this was only happening when running the app in the simulator.

### CODE REVIEW

File: TalkAloud/AudioSlider.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        addTargets()
+    }
+    
+    private func addTargets() {
+        addTarget(self, action: #selector(didStartScrubbing(_:)), for: .editingDidBegin)
+        addTarget(self, action: #selector(sliderValueChanged(_:)), for: .valueChanged)
+        addTarget(self, action: #selector(didEndScrubbing(_:)), for: .valueChanged)
```
Comment (virgil-adumitroae) [inline TalkAloud/AudioSlider.swift]: You are using .valueChanged for both sliderValueChanged and didEndScrubbing.

File: TalkAloud/TalkAloud/ViewControllers/AudioPlayerViewController.swift [hunk 2]
```diff
@@ -154,9 +155,30 @@ class AudioPlayerViewController: UIViewController, AudioEngineStateChangeDelegat
         }
     }
     
+
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/ViewControllers/AudioPlayerViewController.swift:158]: Please remove this empty space

**Review by virgil-adumitroae — CHANGES_REQUESTED** [review]

File: TalkAloud/TalkAloud/ViewControllers/AudioPlayerViewController.swift (hunk 2 above)
Comment (JB184351) [reply TalkAloud/TalkAloud/ViewControllers/AudioPlayerViewController.swift:158]: Done

File: TalkAloud/AudioSlider.swift (hunk 1 above)
Comment (JB184351) [reply TalkAloud/AudioSlider.swift]: Yeah, having didEndScrubbing wouldn't let my audioTime change based on the slider value at this point, I'll look to see what I can do though.

Comment (JB184351) [reply TalkAloud/AudioSlider.swift]: According to the docs the methods `editingDidBegin` and `editingDidEnd` are only applied when changes are being made in a UITextField.

Comment (virgil-adumitroae) [reply TalkAloud/AudioSlider.swift]: Need to find a better solution then if you want to have proper scrubbing :)

**Pull request merged**